### ISSUE TITLE
Package 'nse' has been archived on CRAN for more than 60 days

### ISSUE THREAD

**Issue by statmath-it** [cran-task-views/Econometrics#14]
Package [nse](https://CRAN.R-project.org/package=nse) is currently listed in CRAN Task View [Econometrics](https://CRAN.R-project.org/view=Econometrics) but the package has actually been archived for more than 60 days on CRAN.  Often this indicates that the package is currently not sufficiently actively maintained and should be excluded from the task view.

Alternatively, you might also consider reaching out to the authors of the package and encourage (or even help) them to bring the package back to CRAN.

In any case, the situation should be resolved in the next four weeks. If the package does not seem to be brought back to CRAN, please exclude it from the task view.

**Comment by zeileis** [comment]
Thanks for the reminder. I had reached out to the authors immediately after the CRAN archival, see: https://github.com/keblu/nse/issues/8

The fix would be completely straightforward: Update the maintainer e-mail address (due to changing jobs/universities), rebuild, resubmit. But it is not clear to me whether the authors are planning to do so.

I followed up in the issue linked above. If there is no action in the next weeks we will remove the package from the task view.
